CHANGELOG.rst

```diff
@@ -6,12 +6,18 @@ This document describes changes between each past release.
 6.0.0 (unreleased)
 ------------------
 
+**Breaking changes**
+
+**PostgreSQL 9.5**
```

Comment:
Shouldn't we be more explicit like `PostgreSQL 9.5 is now required for the cache backend`?